ai21_tokenizer/jurassic_tokenizer.py

```diff
@@ -174,6 +174,13 @@ def decode(self, token_ids: List[int], **kwargs) -> str:
         """
         Transforms token ids into text
         """
+        res_text, offsets = self.decode_with_offsets(token_ids)
```

Comment:
if `offsets` is not used in the response, you can do this instead `decoded_text, _ = ...`